config/development.yml.erb

```diff
@@ -70,3 +70,6 @@ mapbox_upload_tileset:    'censustiles-dev'
 
 # Enables DCDO settings via cookies
 use_cookie_dcdo: true
+
+# aiproxy credentials
+aiproxy_api_key: !Secret
```

Comment:
please revert changes to this file, as well as other non-test environments below. this will be inherited from config.yml.erb. for how it works, please see https://github.com/code-dot-org/code-dot-org/blob/staging/config/README.md